### PR TITLE
feat(settings): settings page, monorepo, redeem and gate updates

### PR DESCRIPTION
**Pull request by maurerbot**
# Description

This PR does the following:
- introduces "Galaxy", the GraphQL stiching layer for logically decentralized services
- settings page for the 3ID app
- new redeem app flow for proof only claims
- remove NFT gate in favor of proof gate in 3ID app
- mono repo setup with swtich over to yarn

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires or is a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation website
- [ ] I have made corresponding changes to the literal docs
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: galaxy/src/schemas.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    enum ThreeIDAddressType {
+      ENS
+      ETHEREUM
+      EMAIL
+    }
+
+    enum ThreeIDAddressVisibility {
+      PUBLIC
+      PRIVATE
+      PROTECTED
```
Comment (szkl) [inline galaxy/src/schemas.ts:21]: Not defined yet

**Review by szkl — COMMENTED** [review]
Looks good so far.

**Review by szkl — COMMENTED** [review]
What's the difference between a package and a project?

`console`, `oort`, `redeem`, `starbase`, and, `threeid` are still top-level directories.

**Review by szkl — APPROVED** [review]

File: packages/storybook/.storybook/main.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "../stories/**/*.stories.mdx",
+    "../stories/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions"
+  ],
+  "framework": "@storybook/react"
+}
```
Comment (Cosmin-Parvulescu) [inline packages/storybook/.storybook/main.js:12]: Newlines

File: packages/storybook/.storybook/preview.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+}
```
Comment (Cosmin-Parvulescu) [inline packages/storybook/.storybook/preview.js:9]: Newlines

File: packages/storybook/package.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "start-storybook -p 6006",
+    "build-storybook": "run-p \"build-storybook\"",
+    "dev": "echo \"No dev for storybook. Please use start.\""
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
```
Comment (Cosmin-Parvulescu) [inline packages/storybook/package.json:15]: Fixed versions

File: packages/storybook/package.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "devDependencies": {
```
Comment (Cosmin-Parvulescu) [inline packages/storybook/package.json:19]: Fixed versions

**Review by Cosmin-Parvulescu — APPROVED** [review]
Scared but approving.

**Pull request merged**